docs/release_notes/v0.9.0.md

```diff
@@ -0,0 +1,333 @@
+
+**Refactored** JSONSecretStore to LocalSecretStore (Thanks @cmendible)
+
+**Fixed** the minimum delay of 1s for service invocation
+
+
+### Java SDK
+
+**Fixed** content-type for Spring controller
+
+**Updated** use of TypeRef for deserialization and client APIs
+
+**Updated** samples to use --components-path
+
+**Fixed** documentation for samples directory
+
+
+### .NET SDK
+
+**Updated** dotnet sdk in github actions
+
+**Fixed** actor method names in getting started guide (Thanks @@LayZeeDK)
+
+**Fixed** InvokeAsync example in getting started guide (Thanks @@LayZeeDK)
+
+
+### Python SDK
+
+**Updated** Sphinx documentation (Thanks @chinzhiweiblank)
+
+**Refactored** and split Flask_Dapr from Dapr (Thanks @chinzhiweiblank)
+
+**Refactored** and moves flask_dapr to ext and clean up
+
+**Added** K8s example for actor
+
+
+### Rust/Go SDK
+
+**Added** pub sub example (Thanks @gdhuper)
+
+**Updated** and Beautified code snippet (Thanks @franpog859)
+
+
+### JS-SDK
+
+**Updated** workflow to publish npm package
+
+**Updated** package.json of samples
+
+
+### Docs
+
+**Added** threat model png
+
+**Added** readme note for uninstall in linux
+
+**Updated** minor style guide changes to hybrid-clusters howto stub
+
+**Fixed** typo in setup-azure-servicebus.md (Thanks @candreso)
+
+**Added** docs for dapr init without docker
+
+**Updated** Windows support message into a note
+
+**Updated** cosmos documentation for transaction changes
+
+**Moved** hybrid-cluster guide to howto
+
+**Added** docs for installing dapr to hybrid windows/linux clusters
+
+**Updated** ASB metadata (Thanks @jjcollinge)
+
+**Updated** section related to output binding support
+
+**Updated** documentation related to PostgreSQL (Thanks @brooke-hamilton)
+
+**Updated** bindings_api.md
+
+**Added** cron binding documentation
+
+**Added** binding listing, docs on canceling schedules
+
+**Added** local secret store how to (Thanks @cmendible)
+
+**Added** docs for injecting env vars
+
+**Updated** to rename default config to daprsystem config
+
+**Updated** docs for zipkin init
+
+**Updated** with missing Kafka link in reference section
+
+**Updated** component-secrets.md
+
+**Updated** W3C tracing docs
+
+**Added** Dapr with Docker docs (Thanks @jjcollinge)
+
+**Updated** the doc as how to retrieve the response headers
+
+**Added** How to Use W3C Trace Context
+
+**Added** link from main observability how-to
+
+**Updated** dapr sidecar dashboard and add new actor dashboard
+
+**Added** links to videos from the community calls
+
+**Updated** Event Hubs doc
+
+**Updated** Pub/Sub index
+
+
+## Upgrading to Dapr 0.9.0
+
+If you're upgrading from an older version of Dapr to 0.9.0, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+### Local Machine / Self-hosted
+
+Uninstall Dapr using the CLI you currently have. Note that this will remove the default .dapr directory and the binaries:
```

Comment:
Will also try to remove all containers `dapr_redis, dapr_placement and dapr_zipkin` otherwise will print Warning.